libvast/CMakeLists.txt

```diff
@@ -385,23 +385,30 @@ if (VAST_ENABLE_JOURNALD_LOGGING)
 endif ()
 
 # Link against fmt.
-target_compile_definitions(
-  libvast
-  PUBLIC
-    # Forbid unsafe duration_cast usage.
-    FMT_SAFE_DURATION_CAST
-    # Make fmt::internal an alias for fmt::detail. Remove when requiring fmt >= 7.
-    FMT_USE_INTERNAL)
-find_package(fmt 5.2.1 REQUIRED)
-string(APPEND VAST_FIND_DEPENDENCY_LIST "\nfind_package(fmt 5.2.1 REQUIRED)")
-target_link_libraries(libvast PUBLIC fmt::fmt)
-dependency_summary("fmt" fmt::fmt "Dependencies")
+target_compile_definitions(libvast PUBLIC # Forbid unsafe duration_cast usage.
+                                          FMT_SAFE_DURATION_CAST)
```

Comment:
Do we still need this? Looks like `fmt` has `FMT_SAFE_DURATION_CAST` enabled if not explicitly disabled: https://github.com/fmtlib/fmt/blob/master/include/fmt/chrono.h#L22

Reply:
We probably don't, good catch.